assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -24,14 +25,19 @@
 .component-direct-debit-pop-up-form__content {
   position: relative;
   display: inline-block;
-  top: 50%;
-  width: 345px;
+  width: 305px;
   padding: 10px;
+  @include mq($from: mobileMedium){
+    width: 345px;
+  }
   @include mq($from: mobileLandscape) {
     width: 475px;
   }
+  @media(min-height: 732px) {
```

Comment:
Is there an equivalent of `mq` for height?

Reply:
I think it is fine the way it is, I don't think importing an extra library is worth here.